README.md

```diff
@@ -90,112 +95,246 @@ You can use one of the example frontends available at `./examples` directory.
 
 ### Deploy your frontend
 
-Use netlify or similar and make your frontend public and then let us know by creating a ticket and we will advertise your frontend
-on https://hydra.pretagov.com for others to test.
+Use netlify or similar and make your frontend public.
+Ensure you have correctly set the CORS headers to allow access by the hydra editor.
```

Comment:
@djay Could we add a link to the docs for this (https://6.docs.plone.org/install/containers/images/backend.html?highlight=cors#cors-variables) and/ or a link to our Makefile showing how to set it?

Reply:
This part is about CORS headers for your frontend deployment. CORS for backend is elsewhere